ingestors/npm.go

```diff
@@ -104,15 +100,15 @@ func (ingestor *NPM) Ingest(results chan data.PackageVersion) {
 }
 
 func (ingestor *NPM) SetLatestSequence(seq string) error {
-	err := ingestor.redisClient.Set(context.Background(), NPMLatestSequenceRedisKey, seq, 0).Err()
+	err := RedisClient.Set(context.Background(), NPMLatestSequenceRedisKey, seq, 0).Err()
```

Comment:
@jsonperl just renamed this and imported `redis` as an inline import, instead of `redis.Client`. This way we don't have to wrap all the functions from the core redis library. WDYT?